GLE_solver.py

```
@@ -54,30 +54,31 @@ def boundary_conditions(ya, yb):
     h_a, theta_a, w_a = ya # boundary conditions at s = 0
     h_b, theta_b, w_b = yb # boundary conditions at s = Delta
     return [
-        theta_a - theta0,      # theta(0) = pi/6, this forces theta_a to be essentially theta0. We set. 
+        theta_a - theta0,      # theta(0) = pi/6, this forces theta_a to be essentially theta0. We set.
```

Pylint (3.3.7)

[convention] Line too long (103/100)

(C0301)